### PR TITLE
Astro 1189 update numeric inputs documentation

### PR DESCRIPTION
**Pull request by Design-Kat**
https://rocketcom.atlassian.net/browse/ASTRO-1189

Updated for Numeric Inputs.

### CODE REVIEW

**Comment by Design-Kat** [comment]
@amalyah Could you review prior to me tagging the rest of the team please? Thanks! Kat

File: _content/components/input-field.md [hunk 1]
```diff
@@ -13,19 +13,30 @@ theme: true
...
+- Generally, the size of the text input should match the length of the expected input content.
+- Use help text to provide validation support, rather than placeholder text.
+- Do not use placeholder text for information that is vital to the user's understanding of the text input. A placeholder will disappear once the input has focus and it should only be used for short, clear and generic instructions, such as using "Search..." in a search field.
+- Use required and optional indicators depending on their frequency. e.g. If more fields are optional, only mark required fields.
+- When indicating required fields, an asterisk should be placed to the right of the label.
+- When indicating optional fields, add the word _optional_ in parenthesis to the right of the label.
+- When implementing Numeric inputs, utilize browser defaults whenever possible. Define the range and step size within the HTML element when appropriate. Include the range and step size values in the help text.
+- When appropriate, use input masking to automatically format a user's entry e.g. When entering a phone number, apply a consistent, recognizable format like (XXX) XXX-XXXX or another similar format.
```
Comment (amalyah) [inline _content/components/input-field.md:30]: entry e.g. When > entry. E.g., when

File: _content/components/input-field.md [hunk 2]
```diff
@@ -13,19 +13,30 @@ theme: true
 
 # Input Field
 
-Input Fields allow users to enter text or numeric data.
+Input Fields allow users to enter freeform text. Variations on this field often provide specific data entry formats such as masked data (e.g. passwords or phone numbers), date and time, and numeric data entry.
```
Comment (amalyah) [inline _content/components/input-field.md:16]: Ah. This resolves my concern on the text input version of the page.

File: _content/components/input-field.md [hunk 3]
```diff
@@ -13,19 +13,30 @@ theme: true
...
+- When indicating required fields, an asterisk should be placed to the right of the label.
+- When indicating optional fields, add the word _optional_ in parenthesis to the right of the label.
+- When implementing Numeric inputs, utilize browser defaults whenever possible. Define the range and step size within the HTML element when appropriate. Include the range and step size values in the help text.
+- When appropriate, use input masking to automatically format a user's entry e.g. When entering a phone number, apply a consistent, recognizable format like (XXX) XXX-XXXX or another similar format.
 
 ## Appearance and Behavior
+Text inputs consist of a descriptive, concise label paired with an entry field. Optionally, inputs may include help text, left aligned beneath the input, to assist the user in understanding what kind of content is accepted by the input. Inputs displayed in a form may also be paired with a visual indicator of it's required, or optional state. Disabled inputs are displayed with a different opacity and cannot be interacted with by the user.
```
Comment (amalyah) [inline _content/components/input-field.md:33]: A text input consists of a... 
(Multiple inputs aren't made out of a single label and field.)

left-aligned

File: _content/components/input-field.md [hunk 4]
```diff
@@ -13,19 +13,30 @@ theme: true
...
+
+Standard states for text inputs include Active (the default, interactive state for a text input), Hover (the user has paused over an active or focussed input), Focus (the field is selected and ready for data entry), Disabled (the field is not interactive, and it's content is not sent when the form is submitted), and Read-only.
```
Comment (amalyah) [inline _content/components/input-field.md:35]: its content

File: _content/components/input-field.md [hunk 5]
```diff
@@ -13,19 +13,30 @@ theme: true
...
 
 An Input field is enabled if it is eligible for interaction, and focused if it is the current target for keystrokes.
 
-An input field can be configured for required input, limited data ranges or specific data formats. See [Form and Input Validation](/patterns/forms-and-validation).
+An input field can be configured for required input, limited data ranges or specific data formats. See [Form and Input Validation](/patterns/forms-and-validation). Some variants may allow the user to increment or decrement the value in the control, or to jump to a specific value.
```
Comment (amalyah) [inline _content/components/input-field.md:39]: control or to jump
(remove comma in front of dependent clause)

**Review by amalyah — CHANGES_REQUESTED** [review]
Only small changes on top of the text input PR version.

**Comment by Design-Kat** [comment]
@dmcalester @cskerbo @cheshire89 Tag for review, please! :)

File: _content/components/input-field.md [hunk 6]
```diff
@@ -13,19 +13,30 @@ theme: true
...
 
-Input Fields allow users to enter text or numeric data.
+Input Fields allow users to enter freeform text. Variations on this field often provide specific data entry formats such as masked data (e.g. passwords or phone numbers), date and time, and numeric data entry.
 
 ## Rules of Thumb
 
 - Stacked Input Fields are left justified to one another.
 - Use consistent spacing between stacked fields.
 - Don't use text smaller than what is defined in the CSS. Text size within fields is critical to usability.
+- Generally, the size of the text input should match the length of the expected input content.
```
Comment (amalyah) [inline _content/components/input-field.md:23]: text input > input
(can be true for numeric inputs as well)

File: _content/components/input-field.md [hunk 7]
```diff
@@ -13,19 +13,30 @@ theme: true
...
+- Use help text to provide validation support, rather than placeholder text.
+- Do not use placeholder text for information that is vital to the user's understanding of the text input. A placeholder will disappear once the input has focus and it should only be used for short, clear and generic instructions, such as using "Search..." in a search field.
+- Use required and optional indicators depending on their frequency. e.g. If more fields are optional, only mark required fields.
+- When indicating required fields, an asterisk should be placed to the right of the label.
+- When indicating optional fields, add the word _optional_ in parenthesis to the right of the label.
+- When implementing Numeric inputs, utilize browser defaults whenever possible. Define the range and step size within the HTML element when appropriate. Include the range and step size values in the help text.
```
Comment (amalyah) [inline _content/components/input-field.md:29]: Mixed capitalization on Numeric inputs.

File: _content/components/input-field.md [hunk 8]
```diff
@@ -13,19 +13,30 @@ theme: true
...
+- When appropriate, use input masking to automatically format a user's entry. E.g. When entering a phone number, apply a consistent, recognizable format like (XXX) XXX-XXXX or another similar format.
```
Comment (amalyah) [inline _content/components/input-field.md:30]: E.g., when

File: _content/components/input-field.md [hunk 9]
```diff
@@ -13,19 +13,30 @@ theme: true
...
 
 ## Appearance and Behavior
+Text input consist of a descriptive, concise label paired with an entry field. Optionally, inputs may include help text, left-aligned beneath the input, to assist the user in understanding what kind of content is accepted by the input. Inputs displayed in a form may also be paired with a visual indicator of a required, or optional state. Disabled inputs are displayed with a different opacity and cannot be interacted with by the user.
```
Comment (amalyah) [inline _content/components/input-field.md:33]: Text input consist > An Input Field consists

File: _content/components/input-field.md [hunk 10]
```diff
@@ -13,19 +13,30 @@ theme: true
...
+
+Standard states for text inputs include Active (the default, interactive state for a text input), Hover (the user has paused over an active or focussed input), Focus (the field is selected and ready for data entry), Disabled (the field is not interactive, and its content is not sent when the form is submitted), and Read-only.
```
Comment (amalyah) [inline _content/components/input-field.md:35]: text inputs > input fields

**Review by amalyah — CHANGES_REQUESTED** [review]

File: _content/components/input-field.md (hunk 10 above)
Comment (Design-Kat) [reply _content/components/input-field.md:35]: Also capitalized for consistency with earlier paragraphs.

**Comment by Design-Kat** [comment]
@amalyah updated per your review!

**Review by amalyah — APPROVED** [review]

**Review by dmcalester — APPROVED** [review]

**Pull request merged**